packages/schematics/angular/workspace/files/tslint.json

```diff
@@ -103,15 +103,31 @@
       "check-separator",
       "check-type"
     ],
-    "no-output-on-prefix": true,
-    "use-input-property-decorator": true,
-    "use-output-property-decorator": true,
-    "use-host-property-decorator": true,
+    "angular-whitespace": [true, "check-semicolon", "check-interpolation", "check-pipe"],
+    "banana-in-box": true,
+    "component-class-suffix": true,
+    "contextual-life-cycle": true,
+    "directive-class-suffix": true,
+    "enforce-component-selector": true,
+    "no-attribute-parameter-decorator": true,
+    "no-conflicting-life-cycle-hooks": true,
+    "no-forward-ref": true,
     "no-input-rename": true,
+    "no-output-named-after-standard-event": true,
+    "no-output-on-prefix": true,
     "no-output-rename": true,
+    "no-queries-parameter": true,
+    "pipe-impure": true,
+    "prefer-inline-decorator": true,
+    "prefer-output-readonly": true,
+    "template-conditional-complexity": true,
+    "template-cyclomatic-complexity": [true, 10],
```

Comment:
We should double check if we have a good explanation what the warnings from this issue mean.

Reply:
Complexity based checks can be highly controversial and generally are quite specific to an organization or project.  These are probably best left off by default.